app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -461,6 +461,12 @@ def fraud_result_pending?(enrollment)
       enrollment.profile&.fraud_pending_reason.present?
   end
 
+  # By-pass Primary ID check if enrollment is Enhanced IPP else check for supporting doc type
```

Comment:
Heads-up: For the first test in the testing plan, I had to do the below step
```
Comment out enrollment_outcomes[:enrollments_passed] += 1 in handle_successful_status_update inside app/jobs/get_usps_proofing_results_job.rb
```
before I opened a rails console.

Reply:
I am glad you called that out. I retried one today and wondered if that had always been the case.  It makes sense- if I new-ed up the job and then commented out that line - it would be the old instance. Thank you

 I update the instructions ✅